src/core/ext/filters/client_channel/health/health_check_client.h

```diff
@@ -24,11 +24,11 @@
 #include <grpc/grpc.h>
 #include <grpc/support/sync.h>
 
+#include <atomic>
```

Comment:
Please fix include order to comply with the style guide:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

There are two additional rules specific to C-core:

1. port_platform.h needs to be the first include, in its own group.
2. We have two separate groups of includes for what the style guide calls "your project's headers", one for the angle-bracket includes (such as `<grpc/grpc.h>`) and another for the quoted includes (such as `"src/core/lib/gprpp/arena.h"`).

So in this case, this include should go on line 24, followed by a blank line.

Please fix throughout this PR.

Reply:
Those style guide rules make tooling almost impossible to write.

At the risk of increasing the yak shaving I'm doing this morning, besides the Google import are there any other reasons we write grpc includes with angle-brackets?

Reply:
I don't think there's any other reason, and I'd be fine with not using the angle brackets for our own public headers.  But let's do any such changes in a separate PR. :)

Reply:
So this PR is a reaction to #27014 ... there will be a PR in reaction to this in a few minutes...

Reply:
#27026